UnitTests/ProtectedMockFixture.cs

```diff
@@ -751,6 +771,16 @@ protected virtual string DoReturn(string a, string b)
 			protected virtual void SameFirstParameter(object a) { }
 
 			protected virtual void SameFirstParameter(object a, object b) { }
+
+			protected virtual FooBase OverloadWithDerived(MyBase myBase)
```

Comment:
It's kinda confusing that the method is called `WithDerived` yet it receives a base type. How about naming it simply `Overloaded` or the like?